drivers/hmis/app/views/hmis_admin/project_groups/index.haml

```diff
@@ -0,0 +1,33 @@
+- title = 'HMIS Project Groups'
+= content_for(:title, title)
+%h1= content_for :title
+
+
+- @prompt = "Search project groups..."
+.row.mb-4
+  .col-sm-8
+    = render 'search_form'
+  .col-sm-4.text-right
+    = link_to new_hmis_admin_project_group_path, class: 'btn btn-primary mb-2' do
+      %span.icon-plus
+      Add Project Group
+- if @pagy.count.positive?
+  = render 'common/pagination_top', item_name: 'project group'
```

Comment:
This is fine as is, but it might be worth seeing if `render_paginated_list` could be used easily here.

Reply:
Thanks updated!